Header header
float32 param1
float32 param2
float32 param3
float32 param4
float32 param5
float32 param6
float32 param7
uint16 command
uint8 target_system
uint8 target_component
uint8 confirmation
